Userland/Libraries/LibManual/Node.cpp

```diff
@@ -22,6 +22,16 @@ ErrorOr<NonnullRefPtr<PageNode const>> Node::try_create_from_query(Vector<String
     if (query_parameters.size() > 2)
         return Error::from_string_literal("Queries longer than 2 strings are not supported yet");
 
+    if (query_parameters.size() == 1 && query_parameters[0].starts_with("help://"sv)) {
+        auto help_url = URL::create_with_url_or_path(query_parameters[0].trim("/"sv, TrimMode::Right));
+        auto maybe_node_from_url = Manual::Node::try_find_from_help_url(help_url);
+
+        if (maybe_node_from_url.is_error())
```

Comment:
Can't this be `return *TRY(maybe_node_from_url)->document();`?

Reply:
Yep, that's nicer! I've changed it.